tap_playvox/sync.py

```diff
@@ -44,7 +44,7 @@ def sync_endpoint(client,
     iso_format = "%Y-%m-%dT%H:%M:%S.%fZ"
     start_date = singer.get_bookmark(state,
                                      stream_name,
-                                     client.start_date)
+                                     'endDate')
```

Comment:
```suggestion
                                     key='endDate',
                                     default=client.start_date)
```

Reply:
the conditional on L49 (`if start_date`) returns 2 different formats when doing str(start_datetime), `'2023-08-01T00:00:00.000000Z'` and `'2023-08-01 00:00:00+00:00'`

does it matter which one?